news/2021-06-14-mca-ayim-2019.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Mapper's Choice Awards and A Year In Mapping 2019"
+date: 2021-06-14 00:00:00 +0000
+---
+
+Map connoisseurs rejoice, because the *Mappers' Choice Awards* AND *A Year In Mapping* (AYIM) are back and again and rearing to catch up with the scene!
+
+It's been a long time coming, but these two incredibly popular events are making a grand return in 2021, first focusing on catching up on 2019 during this June and July before moving on to cover all of 2020's maps later on in the year.
+
+With all-new redesigned websites for these two titanic contests, it's never been easier to get involved and have your say on what maps really made the game tick, so make sure to read on below for all the details, dates and everything else you need to become a part of things.
+
+
+## Schedule for MCA and AYIM 2019
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Nomination stage | 2021-06-14 00:00 to 2021-06-21 00:00 |
+| Voting stage | 2021-06-23 00:00 to 2021-07-07 00:00 |
+| Results Stream | 2021-07-10 / 2021-07-11 |
+| Results posted to Website | 2021-07-12 00:00 |
+
+## A Year In Mapping 2019
+
+AYIM is back at long last for another round of highlighting the achievements and statistics of the mapping community in 2019. As in previous editions, each mode will have a section of statistics such as who ranked the most sets, how many difficulties were ranked overall, the extremes in difficulty levels we saw ranked, and many, many more records than you can think of.
+
+**Here's what's new:** In previous years, organizers were responsible for writing comments on each mapper ranked that year. This time, we're opening it up to *everyone*. Know some banger maps from 2019 or a mapper who left a huge impression on you at that time? Let them know your thoughts by posting your comments to the website. This is open for everyone to participate in, so let your positivity be heard!
+
+Writing comments will be open **until** the results of MCA are posted.
+
+All of these statistics and features can be found on the [AYIM Website](https://ayim.corsace.io)
+
+## Mapper's Choice Awards 2019
+
+MCA is an event where eligible users can participate in nominating and voting on the best maps of each of the four game modes as well as storyboards throughout multiple different categories. 
```

Comment:
same deal here too, MCA is *huge* historically and you should at least cover why it was such a big deal and why people might want to care about it